docs/develop/integrations/netlify-integration.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 6
+sidebar_label: Netlify
+title: Integrate Momento With Netlify
+description: Learn to deploy a chat app using Momento Javascript Web SDK to Netlify.
+---
+
+# Integrating Momento with your Netlify app
+
+[Netlify](https://www.netlify.com/) is a popular platform for building, deploying, and scaling modern websites and apps. Their offerings include an advanced global CDN, Git-integrated CI/CD, and free automatic HTTPS support. Their deployment process integrates directly with your Git repositories, making initial deployment about as simple as choosing a repository. Once deployed, updating your sites and apps is handled automatically as you push your changes to Git. 
+
+In this tutorial, we'll make use of a simple example [Next.js chat application](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/web/nextjs-chat) from our [Javascript SDK](https://github.com/momentohq/client-sdk-javascript). By the end of the exercise, you'll have a realtime browser-based chat application up and running at a URL you can use to invite your friends to the chat.
+
+## Prerequisites
+
+To get this app deployed and running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A [Netlify](https://www.netlify.com/) account.
+- A copy or fork of the [Momento Javascript SDK](https://github.com/momentohq/client-sdk-javascript) in your personal repositories.
+- A Momento authentication token.
```

Comment:
is this a pre-req if you show someone how to do it below? IMO, it needs to be one or the other. thoughts?